src/components/order-service/order-service.html

```diff
@@ -75,7 +75,7 @@
         type="button"
         class="btn btn-primary wizard-pf-next"
         ng-if="$ctrl.currentStep.id === 'configure'"
-        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results'"
+        ng-disabled="!$ctrl.selectedProject || $ctrl.currentStep.id === 'results' || $ctrl.invalidNewProject()"
```

Comment:
You can nest forms using `ng-form`.

https://docs.angularjs.org/api/ng/directive/ngForm

I'm suggesting the order service configuration page has a form. The select project component uses ng-form. Then if the select project form is invalid, the outer order service form will be marked invalid. Then we disable the button whenever the form is invalid.

We will need form validation anyway when we add parameters.